syntax = "proto3";

option go_package = "pb/game";

import "common_proto/tip.proto";
option cc_generic_services = true;

message TeamInfoS2CRequest
{
}

message TeamInfoS2CResponse
{
	Tip error = 1;
}


service ClientPlayerTeamService
{
  rpc TeamInfoNotify (TeamInfoS2CRequest) returns (TeamInfoS2CResponse);
}